src/appleseed/renderer/modeling/project/projectfilewriter.h

```diff
@@ -64,11 +57,18 @@ class APPLESEED_DLLSYMBOL ProjectFileWriter
     };
 
     // Write a project to disk. Return true on success, false otherwise.
-    // Write project as project file or as packed project based on options
+    // Write project as project file or as packed project based on options.
     static bool write(
         const Project&  project,
         const char*     filepath,
         const int       options = Defaults);
+
+  private:
+    // Write a project file to disk. Return true on success, false otherwise.
+    static bool write_project_file(
+      const Project&  project,
```

Comment:
Sorry for this indentation issues, I'm setting up indentation setting in IDE so it correspond to your codestyle.
What should be indented with 4 spaces and what with 2?
As I see now:
1. In constructor before ':' it should be 2 spaces
2. In multi-line function invocation it should be 4
3. In multi-line function declaration it should be 4 as well